testplans/lotus-soup/init.go

```diff
@@ -12,10 +12,14 @@ import (
 )
 
 func init() {
-	build.BlockDelaySecs = 2
+	build.BlockDelaySecs = 3
 	build.PropagationDelaySecs = 1
 
-	_ = log.SetLogLevel("*", "WARN")
+	_ = log.SetLogLevel("*", "DEBUG")
+	_ = log.SetLogLevel("dht", "WARN")
+	_ = log.SetLogLevel("swarm2", "WARN")
+	_ = log.SetLogLevel("addrutil", "WARN")
+	_ = log.SetLogLevel("stats", "WARN")
```

Comment:
I've enabled DEBUG logging by default, so that we can monitor TaaS and try to fix flakiness continuously, and also reduced verbosity on some of the louder subsystems.